# Evaluation of full-length 16S rRNA gene sequencing using Oxford Nanopore for diversity surveys of complex, understudied microbiomes
V.Y. Li *1, R. González-Pech 1, V. Román-Reyna 2, D. Lera-Lozano 1, and M. Medina 1  
1 Department of Biology, Eberly College of Science, Pennsylvania State University, University Park, PA, USA  
2 Department of Plant Pathology and Environmental Microbiology, College of Agricultural Sciences, Pennsylvania State University, University Park, PA, USA  
*vivianli@psu.edu  

## References
1. Ames, C. L., Ohdera, A. H., Colston, S. M., Collins, A. G., Fitt, W. K., Morandini, A. C., Erickson, J. S., & Vora, G. J. (2021). Fieldable Environmental DNA Sequencing to Assess Jellyfish Biodiversity in Nearshore Waters of the Florida Keys, United States. Frontiers in Marine Science, 8. https://doi.org/10.3389/fmars.2021.640527
2. Boykin, L. M., Sseruwagi, P., Alicai, T., Ateka, E., Mohammed, I. U., Stanton, J. A. L., Kayuki, C., Mark, D., Fute, T., Erasto, J., Bachwenkizi, H., Muga, B., Mumo, N., Mwangi, J., Abidrabo, P., Okao-Okuja, G., Omuut, G., Akol, J., Apio, H. B., … Ndunguru, J. (2019). Tree lab: Portable genomics for early detection of plant viruses and pests in sub-saharan africa. Genes, 10. https://doi.org/10.3390/genes10090632
3. Carradec, Q., Poulain, J., Boissin, E., Hume, B. C. C., Voolstra, C. R., Ziegler, M., Engelen, S., Cruaud, C., Planes, S., & Wincker, P. (2020). A framework for in situ molecular characterization of coral holobionts using nanopore sequencing. Scientific Reports, 10. https://doi.org/10.1038/s41598-020-72589-0
4. Collins, R. A., Wangensteen, O. S., O’Gorman, E. J., Mariani, S., Sims, D. W., & Genner, M. J. (2018). Persistence of environmental DNA in marine systems. Communications Biology, 1. https://doi.org/10.1038/s42003-018-0192-6
5. El Leithy, A. A., Youssef, A. S. E. D., Nassar, A., Aziz, R. K., Khaled, N. M., Mahrous, M. T., Farahat, G. N., Mohamed, A. H., & Bakr, Y. M. (2025). Long-read 16S rRNA amplicon sequencing reveals microbial characteristics in patients with colorectal adenomas and carcinoma lesions in Egypt. Gut Pathogens, 17. https://doi.org/10.1186/s13099-025-00681-9
6. Epstein, H. E., Brown, T., Akinrinade, A. O., McMinds, R., Pollock, F. J., Sonett, D., Smith, S., Bourne, D. G., Carpenter, C. S., Knight, R., Willis, B. L., Medina, M., Lamb, J. B., Thurber, R. V., & Zaneveld, J. R. (2025). Evidence for microbially-mediated tradeoffs between growth and defense throughout coral evolution. Animal Microbiome, 7, 1. https://doi.org/10.1186/s42523-024-00370-z
7. Erlandson, S. R., Ewing, P. M., Osborne, S. L., & Lehman, R. M. (2024). Sterile sentinels and MinION sequencing capture active soil microbial communities that differentiate crop rotations. Environmental Microbiome, 19. https://doi.org/10.1186/s40793-024-00571-8
8. González-Pech, R. A., Li, V. Y., Garcia, V., Boville, E., Mammone, M., Kitano, H., Ritchie, K. B., & Medina, M. (2024). The Evolution, Assembly, and Dynamics of Marine Holobionts. In Annual Review of Marine Science (Vol. 16, pp. 443–466). Annual Reviews Inc. https://doi.org/10.1146/annurev-marine-022123-104345
9. Heikema, A. P., Horst-Kreft, D., Boers, S. A., Jansen, R., Hiltemann, S. D., de Koning, W., Kraaij, R., de Ridder, M. A. J., van Houten, C. B., Bont, L. J., Stubbs, A. P., & Hays, J. P. (2020). Comparison of illumina versus nanopore 16s rRNA gene sequencing of the human nasal microbiota. Genes, 11, 1–17. https://doi.org/10.3390/genes11091105
10. Hrovat, K., Dutilh, B. E., Medema, M. H., & Melkonian, C. (2024). Taxonomic resolution of different 16S rRNA variable regions varies strongly across plant-associated bacteria. ISME Communications, 4. https://doi.org/10.1093/ismeco/ycae034
11. Klindworth, A., Pruesse, E., Schweer, T., Peplies, J., Quast, C., Horn, M., & Glöckner, F. O. (2013). Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies. Nucleic Acids Research, 41. https://doi.org/10.1093/nar/gks808
12. LaJeunesse, T. C., Parkinson, J. E., Gabrielson, P. W., Jeong, H. J., Reimer, J. D., Voolstra, C. R., & Santos, S. R. (2018). Systematic Revision of Symbiodiniaceae Highlights the Antiquity and Diversity of Coral Endosymbionts. Current Biology, 28(16), 2570-2580.e6. https://doi.org/10.1016/J.CUB.2018.07.008
13. Lawson, C. A., Raina, J. B., Kahlke, T., Seymour, J. R., & Suggett, D. J. (2018). Defining the core microbiome of the symbiotic dinoflagellate, Symbiodinium. Environmental Microbiology Reports, 10(1), 7–11. https://doi.org/10.1111/1758-2229.12599
14. Lin, L., & Ju, F. (2023). Evaluation of different 16S rRNA gene hypervariable regions and reference databases for profiling engineered microbiota structure and functional guilds in a swine wastewater treatment plant. Interface Focus, 13, 20230012. https://doi.org/10.1098/rsfs.2023.0012
15. Maire, J., Girvan, S. K., Barkla, S. E., Perez-Gonzalez, A., Suggett, D. J., Blackall, L. L., & van Oppen, M. J. H. (2021). Intracellular bacteria are common and taxonomically diverse in cultured and in hospite algal endosymbionts of coral reefs. The ISME Journal, 1–15. https://doi.org/10.1038/s41396-021-00902-4
16. Matsuo, Y., Komiya, S., Yasumizu, Y., Yasuoka, Y., Mizushima, K., Takagi, T., Kryukov, K., Fukuda, A., Morimoto, Y., Naito, Y., Okada, H., Bono, H., Nakagawa, S., & Hirota, K. (2021). Full-length 16S rRNA gene amplicon analysis of human gut microbiota using MinIONTM nanopore sequencing confers species-level resolution. BMC Microbiology, 21. https://doi.org/10.1186/s12866-021-02094-5
17. Matthews, J. L., Khalil, A., Siboni, N., Bougoure, J., Guagliardo, P., Kuzhiumparambil, U., DeMaere, M., Le Reun, N. M., Seymour, J. R., Suggett, D. J., & Raina, J. B. (2023). Coral endosymbiont growth is enhanced by metabolic interactions with bacteria. Nature Communications 2023 14:1, 14(1), 1–13. https://doi.org/10.1038/s41467-023-42663-y
18. Oberle, A., Urban, L., Falch-Leis, S., Ennemoser, C., Nagai, Y., Ashikawa, K., Ulm, P. A., Hengstschläger, M., & Feichtinger, M. (2021). 16S rRNA long-read nanopore sequencing is feasible and reliable for endometrial microbiome analysis. Reproductive BioMedicine Online, 42, 1097–1107. https://doi.org/10.1016/j.rbmo.2021.03.016
19. Oliveira, C., Shakiba, E., North, D., McGraw, M., Ballard, E., Barrett-D’amico, M., Glazko, G., & Rahmatallah, Y. (2022). 16S rRNA Gene-Based Metagenomic Analysis of Rhizosphere Soil Bacteria in Arkansas Rice Crop Fields. Agronomy, 12. https://doi.org/10.3390/agronomy12010222
20. Peixoto, R. S., Rosado, P. M., Leite, D. C. de A., Rosado, A. S., & Bourne, D. G. (2017). Beneficial microorganisms for corals (BMC): Proposed mechanisms for coral health and resilience. In Frontiers in Microbiology (Vol. 8, Issue MAR, p. 341). Frontiers Media S.A. https://doi.org/10.3389/fmicb.2017.00341
21. Pollock, F. J., McMinds, R., Smith, S., Bourne, D. G., Willis, B. L., Medina, M., Thurber, R. V., & Zaneveld, J. R. (2018). Coral-associated bacteria demonstrate phylosymbiosis and cophylogeny. Nature Communications, 9. https://doi.org/10.1038/s41467-018-07275-x
22. Ramírez-Amaro, S., Bassitta, M., Picornell, A., Ramon, C., & Terrasa, B. (2022). Environmental DNA: State-of-the-art of its application for fisheries assessment in marine environments. In Frontiers in Marine Science (Vol. 9). Frontiers Media S.A. https://doi.org/10.3389/fmars.2022.1004674
23. Shin, J., Lee, S., Go, M. J., Lee, S. Y., Kim, S. C., Lee, C. H., & Cho, B. K. (2016). Analysis of the mouse gut microbiome using full-length 16S rRNA amplicon sequencing. Scientific Reports, 6. https://doi.org/10.1038/srep29681
24. Stevens, B. M., Creed, T. B., Reardon, C. L., & Manter, D. K. (2023a). Comparison of Oxford Nanopore Technologies and Illumina MiSeq sequencing with mock communities and agricultural soil. Scientific Reports, 13. https://doi.org/10.1038/s41598-023-36101-8
25. Stevens, B. M., Creed, T. B., Reardon, C. L., & Manter, D. K. (2023b). Comparison of Oxford Nanopore Technologies and Illumina MiSeq sequencing with mock communities and agricultural soil. Scientific Reports, 13. https://doi.org/10.1038/s41598-023-36101-8
26. Waechter, C., Fehse, L., Welzel, M., Heider, D., Babalija, L., Cheko, J., Mueller, J., Pöling, J., Braun, T., Pankuweit, S., Weihe, E., Kinscherf, R., Schieffer, B., Luesebrink, U., Soufi, M., & Ruppert, V. (2023). Comparative analysis of full-length 16s ribosomal RNA genome sequencing in human fecal samples using primer sets with different degrees of degeneracy. Frontiers in Genetics, 14. https://doi.org/10.3389/fgene.2023.1213829
27. Yarza, P., Yilmaz, P., Pruesse, E., Glöckner, F. O., Ludwig, W., Schleifer, K. H., Whitman, W. B., Euzéby, J., Amann, R., & Rosselló-Móra, R. (2014). Uniting the classification of cultured and uncultured bacteria and archaea using 16S rRNA gene sequences. Nature Reviews Microbiology, 12, 635–645. https://doi.org/10.1038/nrmicro3330
